extensions/sources/filebase/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ ~  Copyright 2019 Red Hat, Inc.
+ ~
+ ~  Licensed under the Apache License, Version 2.0 (the "License");
+ ~  you may not use this file except in compliance with the License.
+ ~  You may obtain a copy of the License at
+ ~
+ ~    http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~  Unless required by applicable law or agreed to in writing, software
+ ~  distributed under the License is distributed on an "AS IS" BASIS,
+ ~  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~  See the License for the specific language governing permissions and
+ ~  limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    
+    <parent>
+        <groupId>io.smallrye.ext</groupId>
+        <artifactId>smallrye-config-extensions-sources</artifactId>
+        <version>1.0.1-SNAPSHOT</version>
+    </parent>
+    
+    <artifactId>configsource-filebase</artifactId>
```

Comment:
Any reason for `filebase` and not `file`?

Reply:
This is a parent for all file based config sources (json,yaml,properties,xml) so in itself not a config source, but doinf many of the work for these sources. Happy to rename to file, let me know.

Reply:
"Base" just sounds weird to me, I would expect "Abstract" to indicate it's not something you'd use directly.

Anyway, not a huge deal at present

Reply:
So configsource-file-abstract ?

Reply:
Renamed to configsource-file-abstract

Reply:
Renamed package to just ...file